tutorials/text_to_speech.ipynb

```diff
@@ -0,0 +1,283 @@
+                "language = Language(language_code=\"en\")\n",
+                "device = DeviceType.CPU"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Loading Target Audio File\n",
+                "Now let's load and process the audio file that contains the voice we want to target as part of our text-to-speech process. We do segment just the first second of audio since that contains 1 speaker only. "
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "audio = Audio.from_filepath(\"../src/tests/data_for_testing/audio_48khz_mono_16bits.wav\")\n",
+                "ground_truth = \"This is Peter.\"\n",
+                "audio = extract_segments([(audio, [(0.0, 1.0)])])[0][0]"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Preprocessing\n",
+                "Let's preprocess the audio data to make it suitable with the TTS model characteristics that we can find in the model card in the HuggingFace Hub. In particular, for our example model we need the audio to be sampled at 24kHz. "
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "audio = downmix_audios_to_mono([audio])[0]\n",
+                "audio = resample_audios([audio], 24000)[0]"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "And here is how it sounds our target audio."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "play_audio(audio)\n",
+                "print(\"Ground truth:\", ground_truth)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Synthesis\n",
+                "Let's finally synthetize the audio. \n",
+                "\n",
+                "Note: If you want to specify more params and customize the process, you can do it. For more details, see the [**dedicated documentation**](https://sensein.group/senselab/senselab/audio/tasks/text_to_speech.html)."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "res = synthesize_texts(texts=[\"Hello, world. It's nice to meet you.\"], \n",
```

Comment:
This line took 14 minutes to generate, is that expected?

Reply:
i think that's about what I had when running on CPU. don't think there is much that can be done about that.

Reply:
fyi, it took 20 seconds on my laptop

Reply:
do you have a GPU? On GPU/running a large batch of varied audios, I averaged about 25 seconds per audio. This isn't bad but will be hard to scale with, especially since we haven't been able to show performance improvements by splitting across multiple GPUs

Reply:
I have no GPU on my macbook and I request CPU as a preferred device

Reply:
then to be honest I have no idea how it ran so fast since @900miles and I have both used the code and found it can take up to 15 minutes.

Reply:
@wilke0818 sorry, 20 seconds was bark-small. mars5tts takes 8 seconds for the initialization and 4 minutes and 36 seconds for synthesizing "Hello, world. It's nice to meet you." on the CPU

Reply:
Hmm.. Perhaps a warning that it might take a while? So users don't think something is broken after waiting 5–15 minutes.

Reply:
I'm not sure why yours is so much faster. I ran it on openmind, I don't know what kind of cpus they have

Reply:
M2